pyccl/tracers.py

```diff
@@ -533,15 +638,21 @@ class CMBLensingTracer(Tracer):
             The kernel is quite smooth, so usually O(100) samples
             is enough.
     """
-    def __init__(self, cosmo, z_source, n_samples=100):
+    def __init__(self, cosmo, z_source, n_samples=100, k=None):
```

Comment:
Same comment as for `WeakLensingTracer`: explicit k dependence should be avoided

Reply:
Done.